README.md

```diff
@@ -193,3 +193,86 @@ containers used during the `docker build` process. On macOS in particular the
 default is 2GB, which is not enough to build p4c. Increase the memory limit to
 at least 4GB via Docker preferences or you are likely to see "internal compiler
 errors" from gcc which are caused by low memory.
+
+# Known issues
+
+The P4C compiler is in early development. Issues with the compiler are
+tracked on [GitHub](https://github.com/p4lang/p4c/issues). Before
+opening a new issue, please check whether a similar issue is already
+opened. Opening issues and submitting a pull request with fixes for
+those issues is much appreciated.
+
+In addition to the list of issues on Github, there are a number of
+currently unsupported features listed below:
+
+## Frontend
+
+### P4_14 features not supported in P4_16
+
+* extern/blackbox attributes -- there is support for carrying them in
+the IR, but they are lost if P4_16 code is output.  Backends can
+access them from the IR
+
+* Nonstandard extension primitives from P4_14
+  * Execute_meter extra arguments
+  * Recirculate variants
+  * Bypass_egress
+  * Sample_ primitives
+  * invalidate
+
+* No support for P4_14 parser exceptions.
+
+## Backends
+
+### Bmv2
+* All checksum verification happens in the parser.
+* Range match types in tables
+* Tables with multiple apply calls
```

Comment:
@mbudiu-vmw What do you see lacking in terms of missing semantics in P4-14 for tables with multiple apply calls?  In the absence of direct counters and meters, at least, the desired behavior seems fairly clear to me -- repeat construction of a search key, the match operation, and repeat whatever action is found (which could be a different action than happened in a previous apply on the same table, because the search key contents could be different).

Even in the presence of side effects like counters and meters, repeating those sounds like the straightforward desired thing to do.